JS Basics/Objects and Array methods/Problem_6/index.js

```diff
@@ -0,0 +1,26 @@
+function group(arr, criteria) {
+    const groupedPeople = {};
+
+    for(const person of arr) {
+        const groupKey = person[criteria];
+
+        if(!groupedPeople[groupKey]) {
+            groupedPeople[groupKey] = [];
+        }
+
+        groupedPeople[groupKey].push(person);
+    }
+    return groupedPeople
+}
```

Comment:
Implement same function using array.reduce
Do it in the same file and in the same branch